samples/runtimeDI/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
Did you intend to commit Eclipse settings ? If so I think we should also commit for the main project and the `compile-timeDI` sample project